sdk-dotnet/LittleHorse.Sdk/LHInputVariables.cs

```diff
@@ -0,0 +1,114 @@
+        internal string? LHC_OAUTH_CLIENT_ID { get; set; }
+        internal string? LHC_OAUTH_CLIENT_SECRET { get; set; }
+        internal string? LHC_OAUTH_ACCESS_TOKEN_URL { get; set; }
+        internal int LHW_NUM_WORKER_THREADS { get; set; } = 8;
+        internal string LHW_TASK_WORKER_VERSION { get; set; } = string.Empty;
+
+
+        internal LHInputVariables()
+        {
+            var apiHost = Environment.GetEnvironmentVariable("LHC_API_HOST");
+            if (!string.IsNullOrEmpty(apiHost))
+                LHC_API_HOST = apiHost;
+            var apiPort = Environment.GetEnvironmentVariable("LHC_API_PORT");
+            if (!string.IsNullOrEmpty(apiPort))
+                LHC_API_PORT = IntegerConverter.FromString(apiPort);
+            var apiProtocol = Environment.GetEnvironmentVariable("LHC_API_PROTOCOL");
+            if (!string.IsNullOrEmpty(apiProtocol))
+                LHC_API_PROTOCOL = apiProtocol;
+            var clientId = Environment.GetEnvironmentVariable("LHC_CLIENT_ID");
+            if (!string.IsNullOrEmpty(clientId))
+                LHC_CLIENT_ID = clientId;
+            var caCert = Environment.GetEnvironmentVariable("LHC_CA_CERT");
+            if (!string.IsNullOrEmpty(caCert))
+                LHC_CA_CERT = caCert;
+            var clientCert = Environment.GetEnvironmentVariable("LHC_CLIENT_CERT");
+            if (!string.IsNullOrEmpty(clientCert))
+                LHC_CLIENT_CERT = clientCert;
+            var clientKey = Environment.GetEnvironmentVariable("LHC_CLIENT_KEY");
+            if (!string.IsNullOrEmpty(clientKey))
+                LHC_CLIENT_KEY = clientKey;
+            var oauthClientId = Environment.GetEnvironmentVariable("LHC_OAUTH_CLIENT_ID");
+            if (!string.IsNullOrEmpty(oauthClientId))
+                LHC_OAUTH_CLIENT_ID = oauthClientId;
+            var oauthClientSecret = Environment.GetEnvironmentVariable("LHC_OAUTH_CLIENT_SECRET");
+            if (!string.IsNullOrEmpty(oauthClientSecret))
+                LHC_OAUTH_CLIENT_SECRET = oauthClientSecret;
+            var oauthAccessTokenUrl = Environment.GetEnvironmentVariable("LHC_OAUTH_ACCESS_TOKEN_URL");
+            if (!string.IsNullOrEmpty(oauthAccessTokenUrl))
+                LHC_OAUTH_ACCESS_TOKEN_URL = oauthAccessTokenUrl;
+            var numberWorkerThreads = Environment.GetEnvironmentVariable("LHW_NUM_WORKER_THREADS");
+            if (!string.IsNullOrEmpty(numberWorkerThreads))
+                LHW_NUM_WORKER_THREADS = IntegerConverter.FromString(numberWorkerThreads);
+            var taskWorkerVersion = Environment.GetEnvironmentVariable("LHW_TASK_WORKER_VERSION");
+            if (!string.IsNullOrEmpty(taskWorkerVersion))
+                LHW_TASK_WORKER_VERSION = taskWorkerVersion;
+        }
+        
+        internal LHInputVariables(string filePath)
+        {
+            IConfigurationBuilder builder = new ConfigurationBuilder()
+                .AddIniFile(filePath, false, true);
```

Comment:
I believe third argument should be false because we don't want to change these config values if the file changes.

Reply:
done